src/modules/admin/menu/MenuFormSection.tsx

```diff
@@ -1,17 +1,173 @@
+import Image from "next/image";
+import React, { useState } from "react";
+
+type FieldType = {
+  name: string;
+  price: number;
+  category?: Category;
+  ingredients?: string;
+  description?: string;
+  imageURL?: string;
+};
 
 type MenuFormSectionProps = {
   menuLayoutVariant: MenuLayoutVariant;
 };
 
-const MenuFormSection: React.FC<MenuFormSectionProps> = () => {
+const MenuFormSection: React.FC<MenuFormSectionProps> = ({
+  menuLayoutVariant,
+}) => {
+  const [form] = Form.useForm();
+  const [imageURL, setImageURL] = useState<string>(
+    "https://source.unsplash.com/random/?food&plate&11",
+  );
+
+  if (menuLayoutVariant === "edit") {
+    // TODO: get menu data from api then set value to form
+  }
+
+  const onFormSubmit = (values: any) => {
+    // TODO: send data to api
+    console.log(values);
+  };
+
+  const onSave = () => {
+    form.submit();
+  };
+
+  const onCancel = () => {
+    // TODO: popup confirm
+    // TODO: if no close popup
+    // TODO: if yes redirect to menu
+  };
+
+  const onDelete = () => {
+    // TODO: popup confirm
+    // TODO: if no close popup
+    // TODO: if yes delete menu redirect to menu
+  };
+
+  const title = menuLayoutVariant === "edit" ? "Edit" : "Add";
+
   return (
-    <MenuFormCard title={<H4>MenuForm</H4>} bordered={false}>
-      <WireFrame contentNode="AddOnTable" cardColor="red" />
+    <MenuFormCard
+      title={<H4 medium={true}>{title}</H4>}
+      bordered={false}
+      extra={
+        <ButtonGroup>
+          <Button type="primary" onClick={onSave}>
+            Save
+          </Button>
+          <Button type="default">Cancel</Button>
+          {menuLayoutVariant === "edit" && (
+            <Button type="primary" danger>
+              Delete
+            </Button>
+          )}
+        </ButtonGroup>
+      }
+    >
+      <MenuFormContainer
+        layout="vertical"
+        form={form}
+        onFinish={onFormSubmit}
+        requiredMark="optional"
+      >
+        <GeneralFormItemsContainer>
+          <Form.Item<FieldType>
+            name="name"
+            label="Name"
+            rules={[{ required: true, message: "Please input name!" }]}
+            style={{ width: "100%" }}
+          >
+            <Input placeholder="Name" />
+          </Form.Item>
+
+          <Form.Item<FieldType>
+            name="price"
+            label="Price"
+            rules={[{ required: true, message: "Please input price!" }]}
+            style={{ width: "100%" }}
+          >
+            <Input placeholder="50" />
+          </Form.Item>
+
+          <Form.Item<FieldType>
+            name="category"
+            label="Category"
+            style={{ width: "100%" }}
+          >
+            <Select placeholder="Breakfast" allowClear>
+              {categories.map((category) => (
+                <Select.Option value={category.name}>
+                  {category.name}
+                </Select.Option>
+              ))}
```

Comment:
Add key (@minza55113151  probably didn't install ESLINT vs code extension🤔)

Reply:
Thx for ESLINT.
Done.